sdk/eventhub/event-hubs/samples/eventProcessor.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  EventHubClient,
+  EventData,
+  EventPosition,
+  delay,
+  EventProcessor,
+  PartitionContext
+} from "@azure/event-hubs";
+
+class EventProcessorHost {
```

Comment:
`SimplePartitionProcessor` ?

Reply:
Also, we should add the constructor that stores the partition context and then in `processEvents` use the partitionId and consumer group name in the console.log()

This way, the user will know how to get the "partition" related info when they process events

Reply:
Updated!!